<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintainerView</class>
 <widget class="QDialog" name="MaintainerView">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1500</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>系统管理员</string>
  </property>
  <widget class="QGroupBox" name="gbMenu">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>225</y>
     <width>700</width>
     <height>650</height>
    </rect>
   </property>
   <property name="title">
    <string>菜单管理</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <widget class="QTableView" name="tbMenu">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>600</width>
      <height>450</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectItems</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>35</number>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btnAddMenu">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>550</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>添加菜品</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelMenu">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>550</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>删除菜品</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveMenu">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>550</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>保存改动</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUndoMenu">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>550</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>撤销改动</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbAccount">
   <property name="geometry">
    <rect>
     <x>775</x>
     <y>175</y>
     <width>700</width>
     <height>700</height>
    </rect>
   </property>
   <property name="title">
    <string>用户管理</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <widget class="QComboBox" name="cbCata">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblCata">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>50</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>选择用户类别：</string>
    </property>
   </widget>
   <widget class="QTableView" name="tbUser">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>600</width>
      <height>450</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectItems</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblCommit">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>50</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddUser">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>600</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>添加用户</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelUser">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>600</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>删除用户</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveUser">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>600</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>保存改动</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUndoUser">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>600</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>撤销改动</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblInfo">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>175</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnComment">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>175</y>
     <width>125</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>评论管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOrder">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>175</y>
     <width>125</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>订单管理</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMaintainerBGN">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>900</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lblMaintainerBGN</zorder>
  <zorder>gbMenu</zorder>
  <zorder>gbAccount</zorder>
  <zorder>lblInfo</zorder>
  <zorder>btnComment</zorder>
  <zorder>btnOrder</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnSaveMenu</sender>
   <signal>clicked()</signal>
   <receiver>MaintainerView</receiver>
   <slot>SaveChangedMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>697</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>680</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnUndoMenu</sender>
   <signal>clicked()</signal>
   <receiver>MaintainerView</receiver>
   <slot>RecoverMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>693</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>680</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAddMenu</sender>
   <signal>clicked()</signal>
   <receiver>MaintainerView</receiver>
   <slot>NewCuisine()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>626</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>611</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDelMenu</sender>
   <signal>clicked()</signal>
   <receiver>MaintainerView</receiver>
   <slot>DelCuisine()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>608</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>609</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbCata</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MaintainerView</receiver>
   <slot>BindCatalogue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>743</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>995</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbMenu</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>MaintainerView</receiver>
   <slot>OnMenuSelected(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbUser</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>MaintainerView</receiver>
   <slot>OnUserSelected(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>627</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAddUser</sender>
   <signal>clicked()</signal>
   <receiver>MaintainerView</receiver>
   <slot>NewUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>626</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>612</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDelUser</sender>
   <signal>clicked()</signal>
   <receiver>MaintainerView</receiver>
   <slot>DelUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>818</x>
     <y>608</y>
    </hint>
    <hint type="destinationlabel">
     <x>1003</x>
     <y>611</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSaveUser</sender>
   <signal>clicked()</signal>
   <receiver>MaintainerView</receiver>
   <slot>SaveChangedUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>647</x>
     <y>683</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>733</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnUndoUser</sender>
   <signal>clicked()</signal>
   <receiver>MaintainerView</receiver>
   <slot>RecoverUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>847</x>
     <y>693</y>
    </hint>
    <hint type="destinationlabel">
     <x>996</x>
     <y>695</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnComment</sender>
   <signal>clicked()</signal>
   <receiver>MaintainerView</receiver>
   <slot>OnCommentClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>662</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>700</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOrder</sender>
   <signal>clicked()</signal>
   <receiver>MaintainerView</receiver>
   <slot>OnOrderClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>597</x>
     <y>-14</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnMenuSelected(QModelIndex)</slot>
  <slot>SaveChangedMenu()</slot>
  <slot>RecoverMenu()</slot>
  <slot>NewCuisine()</slot>
  <slot>DelCuisine()</slot>
  <slot>BindCatalogue(int)</slot>
  <slot>OnUserSelected(QModelIndex)</slot>
  <slot>NewUser()</slot>
  <slot>DelUser()</slot>
  <slot>SaveChangedUser()</slot>
  <slot>RecoverUser()</slot>
  <slot>OnCommentClicked()</slot>
  <slot>OnOrderClicked()</slot>
 </slots>
</ui>
